exam/src/main/java/in/testpress/exam/ui/ReviewQuestionsFragment.java

```diff
@@ -202,15 +202,8 @@ protected void onLoadFinished() {
                 }
                 setDifficulty(view);
                 progressBar.setVisibility(View.GONE);
-                if (commentsUtil == null && loadComments) {
-                    commentsUtil = new CommentsUtil(
-                            ReviewQuestionsFragment.this,
-                            getLoaderManager(),
-                            CommentsUtil.getQuestionCommentsUrl(apiClient, reviewItem),
-                            rootLayout,
-                            ((ReviewQuestionsActivity) getActivity()).buttonLayout
-                    );
-                    commentsUtil.displayComments();
+                if (loadComments) {
```

Comment:
the reason this loadcomments condition was added because since comments are always getting loaded for every review question, we didn't want to so we disabled comments for all institutes and enable only for few. But as comments are going to be loaded only on click in this PR this check is unnecessary.